### PR TITLE
Initial commit for MOZ-SMC

### PR DESCRIPTION
**Pull request by amitverma-egov**
Initial commit for MOZ-SMC

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/blocs/blocs-smc/closed/closed_household.dart [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+      );
+
+      await householdMemberRepository.create(householdMember.copyWith(
+          clientAuditDetails: householdMember.clientAuditDetails!.copyWith(
+        lastModifiedTime: initialModifiedAt,
+      )));
+
+      await taskRepository.create(task);
+
+      // print("Task - ${task}");
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/blocs/blocs-smc/closed/closed_household.dart]: remove the comment

File: apps/health_campaign_field_worker_app/lib/pages/pages-SMC/beneficiary/widgets/consent_household_acknowledgement.dart [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import 'package:auto_route/auto_route.dart';
+import 'package:digit_components/digit_components.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:registration_delivery/blocs/household_overview/household_overview.dart';
+import 'package:registration_delivery/router/registration_delivery_router.gm.dart';
+
+// import '../../../blocs/household_overview/household_overview.dart';
+import '../../../../utils/i18_key_constants.dart' as i18;
+// import '../../../widgets/localized.dart';
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/pages/pages-SMC/beneficiary/widgets/consent_household_acknowledgement.dart]: remove the comments

File: apps/health_campaign_field_worker_app/lib/pages/pages-SMC/beneficiary/custom_beneficiary_details_smc.dart [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+                                    //               (e) =>
+                                    //                   e.identifierType ==
+                                    //                   IdentifierTypes
+                                    //                       .uniqueBeneficiaryID
+                                    //                       .toValue(),
+                                    //             )
+                                    //             .identifierId ??
+                                    //         localizations.translate(
+                                    //           i18.common.noResultsFound,
+                                    //         ),
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/pages/pages-SMC/beneficiary/custom_beneficiary_details_smc.dart]: rmove the comments

**Review by vishal-egov — CHANGES_REQUESTED** [review]

File: apps/health_campaign_field_worker_app/lib/blocs/blocs-smc/closed/closed_household.dart (hunk 1 above)
Comment (amitverma-egov) [reply apps/health_campaign_field_worker_app/lib/blocs/blocs-smc/closed/closed_household.dart]: removed

**Review by vishal-egov — APPROVED** [review]

**Pull request merged**